### PR TITLE
Improve graph design

### PR DESCRIPTION
**Pull request by benedihm**

### CODE REVIEW

**Review by Kapteinkrok — APPROVED** [review]
It's good that someone takes design seriously:) The colours of the descriptions and the actual graph nodes corresponds now, compared to current main. Additionally, I have added an issue ( #169) for making the "utviklingområde" buttons and "utviklingsområde" graphnodes to be the same. However, this is something to be discussed at a later stage.

File: frontend/src/d3/GraphSimulation.ts [hunk 1]
```diff
@@ -588,7 +590,8 @@ export default class {
         graphEdges
           .selectChild(this.selectNodeOrEdge)
           .transition('500')
-          .attr('stroke-width', edgeWidth / this.scale);
+          .attr('stroke-width', edgeWidth / this.scale)
+          .attr('stroke', '#A0AEC0');
```
Comment (AndreasWintherMoen) [inline frontend/src/d3/GraphSimulation.ts:594]: Replace with edgeColor?

**Review by AndreasWintherMoen — CHANGES_REQUESTED** [review]
Looks good. A few minor code details should be fixed.

File: frontend/src/components/molecules/GraphDescriptions.tsx [hunk 2]
```diff
@@ -4,11 +4,11 @@ import GraphNodeKey from '../atoms/GraphNodeKey';
 
 const GraphDescriptions: React.FC = () => (
   <Stack px="5">
-    <GraphNodeKey color="#4299e1" description="Standardfarge/StartNode" />
-    <GraphNodeKey color="#EA5F41" description="Bærekraftsmål" />
-    <GraphNodeKey color="#85E664" description="Delmål til bærekraftsmål" />
-    <GraphNodeKey color="#4299e1" description="Trippelbunnlinje" />
-    <GraphNodeKey color="#DDA93B" description="Utviklingsområde" />
+    <GraphNodeKey color="#63B3ED" description="Standardfarge/StartNode" />
```
[the remark this answers is not in the document]
Comment (benedihm) [reply frontend/src/components/molecules/GraphDescriptions.tsx]: Indeed, fixed!

File: frontend/src/d3/GraphSimulation.ts [hunk 3]
```diff
@@ -545,7 +546,8 @@ export default class {
         graphEdges
           .selectChild(this.selectNodeOrEdge)
           .transition('500')
-          .attr('stroke-width', (edgeWidth * 3) / this.scale);
+          .attr('stroke-width', (edgeWidth * 1.5) / this.scale)
+          .attr('stroke', '#00A3C4');
```
[the remark this answers is not in the document]
Comment (benedihm) [reply frontend/src/d3/GraphSimulation.ts]: Fixed!

File: frontend/src/d3/GraphSimulation.ts (hunk 1 above)
Comment (benedihm) [reply frontend/src/d3/GraphSimulation.ts]: Fixed!

**Review by erikskaar — APPROVED** [review]

**Review by AndreasWintherMoen — APPROVED** [review]
LGTM

**Pull request merged**